src/routes/about.tsx

```diff
@@ -0,0 +1,109 @@
+import Page from '@/components/page'
+import { css } from 'linaria'
+
+const pageStyle = css`
+  text-align: center;
+  display: grid;
+  grid-template-columns: 1fr minmax(60%, 61rem) 1fr;
+  padding: 0 1rem;
+`
+
+const headerStyle = css`
+  grid-column: 2;
+  margin-bottom: 0rem;
+`
+
+const informationStyle = css`
+  font-size: 22px;
+  grid-column: 2;
+`
+
+const sourceStyle = css`
+  padding: 1rem 2rem 0 2rem;
+  font-family: monospace;
+  grid-column: 2;
+`
+
+const boldText = css`
+  font-weight: bold;
+`
+
+const boldSourceText = css`
+  font-weight: bold;
+  margin: 0.2rem;
+`
+
+const sourceText = css`
+  margin: 0.2rem;
+`
+
+const AboutPage = () => {
+  return (
+    <Page name="About" back="/" class={pageStyle}>
+      <h1 class={headerStyle}>Welcome to Peregrine!</h1>
+      <div class={informationStyle}>
+        <p>
+          <span class={boldText}>
+            Peregrine is a completely free, open-source scouting app for FRC
+            competitions.
+          </span>{' '}
```

Comment:
@varCepheid Can we use semantic HTML elements for text like this instead of putting CSS classes on `<span>`s? [Here's an article explaining how semantic elements work and benefit people using assistive technology](https://blog.hubspot.com/website/semantic-html). Plus, I think we'll end up with less code to do the same thing!

In the specific case of boldText, we can change `<span class={boldText}>` → `<strong>`